src/components/chips/js/chipDirective.js

```diff
@@ -0,0 +1,47 @@
+(function () {
+  'use strict';
+  angular
+      .module('material.components.chips')
+      .directive('mdChip', MdChip);
+
+  /**
+   * @ngdoc directive
+   * @name mdChip
+   * @module material.components.chips
+   *
+   * @description
+   * `<md-chip>` is a component used within `<md-chips>` and is responsible for rendering individual chips.
+   *
+   *
+   * @usage
+   * <hljs lang="html">
+   *   <md-chip>{{$chip}}</md-chip>
+   * </hljs>
+   *
+   */
+
+  var DELETE_HINT_TEMPLATE = '\
+      <span class="visually-hidden">Press delete to remove this chip</span>';
```

Comment:
Still need to do this for this message and the delete button hint. Would it be acceptable to define a default value for these messages and allow them to be overridden via attributes on md-chips?